manifests/container.pp

```diff
@@ -10,21 +10,22 @@
 #   the content base url.
 # @param pulpcore_https_vhost
 #   The name of the Apache https vhost for Pulpcore
+# @param cname
+#   Common name to check for authentication against
 class foreman_proxy_content::container (
   String $location_prefix = '/pulpcore_registry',
   String $registry_v1_path = '/v1/',
   String $registry_v2_path = '/v2/',
   String $pulpcore_https_vhost = 'pulpcore-https',
+  String $cname = $facts['networking']['fqdn'],
```

Comment:
Hmm, should the type be `Stdlib::Fqdn` to match `$certs::foreman_proxy::hostname`?

It might still be nice to have the check for custom hiera or non-installer use of the module.